src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.Unix.cs

```diff
@@ -20,21 +23,33 @@ public sealed partial class RuntimeThread
 
         internal WaitSubsystem.ThreadWaitInfo WaitInfo => _waitInfo;
 
+#if MONO
+        private void Unix_PlatformSpecificInitialize()
```

Comment:
Because I will use in corert/RuntimeThread.cs in corlib. For example: [RuntimeThread](https://github.com/mono/mono/pull/5054/commits/de6ad35c3965297bc5dbe45cdd7d0c2d0a89a593#diff-66cfe549fd788550269a7f45908cbbd1R10)

I applied technique that is used be @luhenry in this PR: [[corert] Import System.Threading.ThreadPoolBoundHandle](https://github.com/mono/mono/commit/b85e7c862b17520961a7e03d83600f3efe19adfe#diff-23b243ed1a5acfc0cfb508adbb2888c9R4)

I think that in corert class layout when RuntimeThread,Unix and RuntimeThread.Windows has methods with equal names (e.g. PlatformSpecificInitialize()) this technique must be applied to resolve symbols redeclareation.